src/js/src/router/index.js

```diff
@@ -44,6 +44,15 @@ const routes = [
     },
     component: () =>
       import(/* webpackChunkName: "page-harvest-data" */ '../views/PageHarvestData.vue')
+  },
+  {
+    path: '/pwid',
+    name: 'PWID',
+    meta: {
+      title: 'PWID'
+    },
+    component: () =>
+      import(/* webpackChunkName: "page-harvest-data" */ '../views/PWID.vue')
```

Comment:
probably the wrong comment? Its no longer "page-harvest-data" ;)

Reply:
Damn you copy/paste - thanx:)